core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -344,7 +344,11 @@ int findVersion() {
 
         return maxVersion;
       } catch (IOException io) {
-        LOG.warn("Error trying to recover version-hint.txt data for {}", versionHintFile, e);
+        LOG.warn(
+            "Error trying to recover {} data for {}",
+            Util.VERSION_HINT_FILENAME,
+            versionHintFile,
```

Comment:
`versionHintFile` object contains `Util.VERSION_HINT_FILENAME` in my understanding. 
How about changing to `"Error trying to recover version hint data for {}"`?

Reply:
Thanks. As you say, it is likely redundant. I am trying to rephrase it.

```
Error trying to recover version-hint.text data for file:/var/folders/0n/191xk_2s2_3dbz7t6_jpws100000gn/T/junit-14873554957427488870/metadata/version-hint.text
```